docs/README.md

```diff
@@ -1,44 +1,9 @@
 # Documentation
 
-The documentation is available at: [sbi-dev.github.io/sbi](http://sbi-dev.github.io/sbi)
-
-## Building the Documentation
-
-We use [`mike`](https://github.com/jimporter/mike) to manage, build, and deploy our
-documentation with [`mkdocs`](https://www.mkdocs.org/). To build the documentation
-locally, follow these steps:
-
-1. Install the documentation dependencies:
-
-    ```bash
-    python -m pip install .[doc]
-    ```
-
-2. Convert the current version of the documentation notebooks to markdown and build the
```

Comment:
how is this handled with sphinx? how are we converting and displaying notebooks in the docs?

Reply:
converting and displaying happens automatically. It also detects which notebooks got changed and only (runs and) converts those.